float64 x
float64 y
float64 z
float64 width
---
bool filled
string[] objects
